README.md

```diff
@@ -36,15 +36,51 @@ pip install -r requirements-min.txt
 export FLASK_DEBUG=1
 ```
 
+### R dependencies
+
+Install [Pandoc](https://pandoc.org/installing.html)
+```sh
+brew install pandoc
+```
+
+```sh
+R
+> install.packages(c("rmarkdown", "stringi", "tidyverse", "Seurat", "ggforce"))
+```
+
 
 Write a config file
 ```sh
 cp config.dev.py config.py
 ```
 
+Ensure hosts file has the following record:
+```
+127.0.0.1       local.dsde-dev.broadinstitute.org
+```
+
+Update main.py to use Broad's wildcard SSL certificates.
+These certificates are the same ones used for any of our web applications.
+To get these certificates, run the `configure.rb` script by following the instructions under the title [Running Leo Locally](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/104399223/Callisto+Developer+Handbook#CallistoDeveloperHandbook-RunningLeoLocally)
+
+Once complete, copy `leonardo/config/server.*` to `/etc/ssl/certs`.
+
+Configure flask to look for the SSL Certificates
+
+```py
+# main.py
+if __name__ == '__main__':
+    app.run(port=8080, host='0.0.0.0', ssl_context=('/etc/ssl/certs/server.crt', '/etc/ssl/certs/server.key'))
+```
+
+Edit config.py to use development authentication.
+``` py
+SAM_ROOT = 'https://sam.dsde-dev.broadinstitute.org'
+```
+
```

Comment:
can you document the url for access calhoun locally somewhere too

Reply:
Added!